source/userguide/custom_action.rst

```diff
@@ -0,0 +1,219 @@
+Working with Custom Action
+==========================
+
+MindMeld 4.3 introduces Custom Actions which allow MindMeld developers to further separate their business
```

Comment:
```suggestion
MindMeld 4.3 provides the ability for external applications to integrate custom dialogue management logic with MindMeld applications.
```

Reply:
This line actually wraps to the next line.

Reply:
Sorry I dont understand. Why can't we incorp these changes?

Reply:
The rendered line will be: "MindMeld 4.3 introduces Custom Actions which allow MindMeld developers to further separate their business logic from the conversational application."

Basically, it will combine the two sentences together.